sympy/stats/frv_types.py

```diff
@@ -401,11 +494,15 @@ def Hypergeometric(name, N, m, n):
 
 
 class RademacherDistribution(SingleFiniteDistribution):
+
     @property
-    @cacheit
-    def dict(self):
-        return {-1: S.Half, 1: S.Half}
+    def set(self):
+        return [-1, 1]
```

Comment:
this is not a set

Reply:
I will make the changes across other distributions to return `set`.